ui/docs/manual/using/artifacts.mdx

```diff
@@ -9,8 +9,10 @@ import Warning from 'taskcluster-ui/views/Documentation/components/Warning';
 
 ## Creating Artifacts
 
-Most workers automatically produce a conventional `public/logs/live.log`
-artifact containing the main log of the artifact.
+Workers conventionally create a `public/logs/live.log`
+artifact containing the main task log. This default path can be overridden
+in Generic Worker task payloads using the `logs.live` property and the
+`log` property for Docker Worker task payloads.
```

Comment:
stupid question, if you define `logs.live` would it replace whole path, or just the last part of it? 
So if `logs.live: 'non-public/banana.log'` then artifact path would be `non-public/banana.log`?

Reply:
Your example is correct :)